munkipkg

```diff
@@ -639,6 +639,17 @@ def build(project_dir, options):
             if retcode:
                 raise BuildError("Could not remove existing %s" % outputname)
 
+        # remove .DS_Store file from the scripts folder
+        if os.path.exists(os.path.join(build_info['scripts'], ".DS_Store")):
+            display("Removing .DS_Store file from the scripts folder")
+            os.remove(os.path.join(build_info['scripts'], ".DS_Store"))
+
+        # make scripts executable
+        for pkgscript in ("preinstall", "postinstall"):
+            display("Making %s script executable" % pkgscript)
```

Comment:
Could we do this only if the files are missing the needed attributes?
Something like:
```python
    if hasRequiredMode(os.path.join(build_info['scripts'], pkgscript), 0o555):
```

Where that function is something like:

```python
def hasRequiredMode(pathname, mode):
    return (os.stat(pathname).st_mode & mode) != 0
```

and then we can argue if the "required" mode should be 0x555 or 0x500...

But would like to remove output "noise" if the file has the correct attributes to start with.